### ISSUE TITLE
nedit-ng does not open files located inside a directory whose name contains accents.

### ISSUE THREAD

**Issue by ericxuo** [eteran/nedit-ng#22]
Hi,

nedit-ng can't open a file located inside (for example) : /home/xuo/Téléchargements
nc-ng kodi.log
![capture d ecran_2017-12-29_14-56-38](https://user-images.githubusercontent.com/9089499/34438629-809d2642-eca8-11e7-88de-2416ba54bdd8.png)

If I copy the file in /home/xuo, then I've got no problem.
I think the issue comes from the fact that "Téléchargements" contains letters with accents.
Some informations : 
/home/xuo/Téléchargements 217 # printenv | grep LANG
LANGUAGE=fr_FR.UTF-8:fr
LANG=fr_FR.UTF-8

Regards.

Xuo.

**Comment by eteran** [comment]
Interesting. Do you happen to know if the original nedit had any issue with this? I actually would expect my usage of QStrings internally to have made nedit-ng's support for non-english setups **better**. I'll look into it.

**Comment by eteran** [comment]
OK, I was able to reproduce this. I'll look into which code paths cause the error

**Comment by eteran** [comment]
I believe the latest push should fix this issue. Conversion of paths temporarily in latin1 strings was causing the problem. Ideally, I will be able to replace with functions with ones that don't temporarily convert, but for now this should work OK.

**Comment by ericxuo** [comment]
Hi,

I can't compile nedit-ng anymore.
I've got the following error : 
CMake Error at /lib64/cmake/Qt5Gui/Qt5GuiConfigExtras.cmake:9 (message):
  Failed to find "GL/gl.h" in "/usr/include/libdrm".
At the moment, I don't know how to solve this issue. It does not come from nedit-ng as I can't compile the rc6 tag neither.
I changed my graphics card. I don't see the relationship.
What I mean is that it will take me some time to check the accent issue has been corrected.

Regards.

Xuo.
